subsys/spm/Kconfig

```diff
@@ -96,7 +96,7 @@ config SPM_SERVICE_RNG
 	bool "Request random numbers"
 	default y if SPM
 	select NORDIC_SECURITY_BACKEND if IS_SPM
-	select FPU if IS_SPM
+	select FPU
```

Comment:
Is this really needed @sigvartmh ?
Would it be better to just remove FPU selection completely, so it is not present in the application ? Why would it be tight to the RNG Service?

Reply:
Guess, if the application enables FPU, the rest of the stuff here will be able to catch the missmatch.
If it is essential that the RNG service enables the FPU in NS, this should be documented here, IMHO.

Reply:
I would recommend a selection based on what the NS configuration states as well as manual stacking of FPU registers (I think we do this for TF-M). I don't think FPU usage inside SPM is a very common use-case so the problem to solve is more on the EABI rather than anything